mvrx/src/main/kotlin/com/airbnb/mvrx/MvRxViewModelProvider.kt

```diff
@@ -156,4 +113,38 @@ object MvRxViewModelProvider {
         }
         return null
     }
+
+    /**
+     * For internal use only. Public for inline.
```

Comment:
you can make this `internal` and use `@PublishedApi` instead of needing to make it public